python/src/llvm.cc

```diff
@@ -302,6 +303,21 @@ void init_triton_llvm(py::module &&m) {
         PassBuilder pb(nullptr /*targetMachine*/, tuningOptions, std::nullopt,
                        instrCbPtr);
 
+        std::string pluginFile =
+            mlir::triton::tools::getStrEnv("AMDGCN_INSTRUMENTATION_LIB");
+        if (!pluginFile.empty()) {
+          llvm::errs()
```

Comment:
Missing it should be fine and actually the normal case? Don't think we want mandate every developer to provide the path here?

Reply:
Right, so if the string ENV isn't set we just continue as normal. The conditional could be reversed to make that more clear.